### PR TITLE
Update OpenID IDP form

### PR DESCRIPTION
**Pull request by spadgett**
* Make Client Secret a password field instead of file upload
* Prefill default claim values
* Add form help text

/assign @jcaianirh 
@enj fyi

Add Identity Provider: OpenID Connect · OKD 2019-03-28 14-50-56

### CODE REVIEW

File: frontend/public/components/cluster-settings/openid-idp-form.tsx [hunk 1]
```diff
@@ -223,13 +224,13 @@ export class AddOpenIDPage extends PromiseComponent {
...
         </div>
         <div className="form-group">
-          <DroppableFileInput
-            onChange={this.clientSecretFileChanged}
-            inputFileData={clientSecretFileContent}
-            id="clientsecret-file"
-            label="Client secret"
-            isRequired
-            hideContents />
+          <label className="control-label co-required" htmlFor="clientSecret">Client Secret</label>
```
Comment (jcaianirh) [inline frontend/public/components/cluster-settings/openid-idp-form.tsx]: @spadgett you're losing the drag/drop capability by changing this field, is that what you intended?

File: frontend/public/components/cluster-settings/openid-idp-form.tsx [hunk 2]
```diff
@@ -26,10 +26,10 @@ export class AddOpenIDPage extends PromiseComponent {
     name: 'openid',
     mappingMethod: 'claim',
     clientID: '',
-    clientSecretFileContent: '',
-    claimPreferredUsernames: [],
-    claimNames: [],
-    claimEmails: [],
+    clientSecret: '',
```
Comment (jcaianirh) [inline frontend/public/components/cluster-settings/openid-idp-form.tsx]: similar to htpasswd formatting, do you wanna update that as well?

File: frontend/public/components/cluster-settings/openid-idp-form.tsx [hunk 3]
```diff
@@ -223,13 +224,13 @@ export class AddOpenIDPage extends PromiseComponent {
...
         <div className="form-group">
-          <DroppableFileInput
-            onChange={this.clientSecretFileChanged}
-            inputFileData={clientSecretFileContent}
-            id="clientsecret-file"
-            label="Client secret"
-            isRequired
-            hideContents />
+          <label className="control-label co-required" htmlFor="clientSecret">Client Secret</label>
+          <input className="form-control"
```
Comment (jcaianirh) [inline frontend/public/components/cluster-settings/openid-idp-form.tsx]: @spadgett should this component change exist in both openid and htpasswd?

File: frontend/public/components/cluster-settings/openid-idp-form.tsx (hunk 2 above)
Comment (spadgett) [reply frontend/public/components/cluster-settings/openid-idp-form.tsx]: HTPasswd does actually use a file. Client secret for OpenID is just a string.

File: frontend/public/components/cluster-settings/openid-idp-form.tsx (hunk 1 above)
Comment (jcaianirh) [reply frontend/public/components/cluster-settings/openid-idp-form.tsx]: @spadgett ah got it.

**Pull request merged**